Cubical/Cohomology/EilenbergMacLane/Gysin.agda

```diff
@@ -0,0 +1,1315 @@
+{-# OPTIONS --safe --lossy-unification #-}
+
+{-
+This file contains
+1. The Thom isomorphism (various related forms of it)
+2. The Gysin sequence
```

Comment:
Wouldn't it make sense to split these into two files?

Reply:
The two are so closely related and the module about Thom contains info about the Gysin sequence in a crucial way. I think it would be more confusing/messier to split them up.

Reply:
Ok